### DISCUSSION TITLE
Where do I post bug/issue reports for the Laravel documentation?

### DISCUSSION THREAD

**Discussion by nagmat84** [laravel/framework#42865] [Q&A]
The Githib repository of the Laravel documentation https://github.com/laravel/docs does not allow to open issues or start a discussion. Where do I post bug/issue reports for the Laravel documentation?

**Comment by henzeb** [comment]
where you are now. https://github.com/laravel/framework/issues

**Comment by abdulsalamamtech** [comment]
<img width="1115" height="710" alt="image" src="https://github.com/user-attachments/assets/e5d3d5f7-bb19-46e3-bf97-9939c26934b6" />
Hello, Good day developers!
Please I need a quick fix on this issue I am facing when I tried to optimize or cache the view of my Laravel application.

```
php artisan optimize:clear
php artisan optimize 
# or 
php artisan view:cache
```

I have tried all this options (from Gemini, Copilot etc)😔 
But I'm still getting this same issue.
I think the issue is from Laravel itself due to:


`vendor/laravel/framework/src/Illuminate/View/Compilers/ComponentTagCompiler.php:315`

Thanks!

**Reply by henzeb** [reply]
Did you run composer dump-autoload?

**Reply by abdulsalamamtech** [reply]
Yes, I did that and more.

**Reply by abdulsalamamtech** [reply]
I have been able to trace to issue to Filament pages.
I have commented most of the content on:
```
resources/views/filament/pages/*
```
I will still keep digging.
Thank you all.
